src/components/app/ProfileViewer.js

```diff
@@ -59,8 +81,19 @@ class ProfileViewer extends PureComponent<Props> {
           <div className="profileViewerSpacer" />
           <MenuButtons />
         </div>
-        <Timeline />
-        <DetailsContainer />
+        <SplitterLayout
+          customClassName="profileViewerSplitter"
+          vertical
+          percentage={false}
+          // The DetailsContainer is primary.
+          primaryIndex={1}
+          // The Timeline is secondary.
+          secondaryInitialSize={270}
```

Comment:
Wondering if the initial size couldn't be `timelineHeight`. But I'm not sure. I suppose you thought of it and decided this value instead, what were the cons ?